### ISSUE TITLE
Unable to load picture or PDF file 'https://quarto.org/docs/get-started/hello/images/rstudio-hello.png'.

### ISSUE THREAD

**Issue by johannes4998** [quarto-dev/quarto-web#485]
When I change the format from html to pdf in the getting-started (example)[https://quarto.org/docs/get-started/hello/rstudio.html]

I get the error
```
compilation failed- error
Unable to load picture or PDF file 'https://quarto.org/docs/get-started/hello/images/rstudio-hello.png'.
```
When I save the image local it works but I would prefer using the link.

**Comment by jjallaire** [comment]
Agreed it would be better if this could render unmodified w/ PDF. It isn't because we use a remote reference to the PNG so that the entire example is downloadable as a single file.

**Comment by cscheid** [comment]
Just for what's worth, I can imagine a Lua filter that finds external links, downloads them locally, and rewrites the link to use the extension. (I don't think we want to have this feature in quarto by default, but it should be possible for users to define this kind of behavior externally)
